<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1338</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="program_size_label">
         <property name="text">
          <string>Tamano</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="program_size_edit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="program_name_label">
         <property name="text">
          <string>Nombre</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="program_name_edit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="save_program_button">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>1301</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>20</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Paginas</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="page_amount_edit">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>20</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="page_amount_save_btn">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>70</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setup_btn">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>140</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Setup</string>
    </property>
   </widget>
   <widget class="QComboBox" name="program_combo_box">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>79</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_to_exec_btn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to MMU</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>20</y>
      <width>79</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>FIFO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2da Oportunidad</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LRU</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NRU</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Optimo</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1338</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
